src/Acuminator/Acuminator.Vsix/Commands/DiagnosticSuppression/ProjectSuppressionFileGenerator.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Dynamic;
+using System.Linq;
+using System.Threading;
+using System.IO;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Acuminator.Utilities.Common;
+using Acuminator.Utilities.Roslyn;
+using Acuminator.Utilities.DiagnosticSuppression;
+using Acuminator.Vsix.Utilities;
+
+
+
+namespace Acuminator.Vsix.DiagnosticSuppression
+{
+	/// <summary>
+	/// A generator of supression file for project.
+	/// </summary>
+	public static class ProjectSuppressionFileGenerator
+	{
+		private const string CustomSuppresionFileSuffix = "Custom";
```

Comment:
What if a user wants to edit a shared suppression file?

Reply:
This part is outdated and actually will be changed. We won't implement custom supression file functionality as I though when I wrote this. I' ll delete it later when I start working on add to suppression file command

Reply:
This branch contains some outdated code which will be changed in the future. 
It certainly is not a  finished functionality. I want to preserve part of it because of the wrapper to the Roslyn internal diagnostic service and some other parts. And I don't want to clear it up and finish the "add to suppression file" command right now

Reply:
@SENya1990 so why can't we keep it in a feature branch? I don't see any major conflicts with VS2019 & Roslyn 2 migration. We planned to release 1.6.2, so it is not desired to have some unfinished functionality in the final build.

Reply:
Well, I didn't want to keep very old branches but I think you actually right